### PR TITLE
Updated Byteman & Arquillian to latest version.

### PR DESCRIPTION
**Pull request by dipak-pawar**
1.Updated byteman & arquillian to latest version.
2.Tested it against Wildfly- managed

---
This change is [Reviewable](https://reviewable.io/reviews/arquillian/arquillian-extension-byteman/8)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,370 +1,280 @@
...
+		<profile>
+			<id>jboss as</id>	
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<arq.container.chameleon.configuration.chameleonTarget>jboss
+									as:7.1.1.Final:managed
```
Comment (bartoszmajsak) [inline pom.xml]: Does it work with Java 8?

File: pom.xml [hunk 2]
```diff
@@ -1,370 +1,280 @@
...
+							</rules>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+
+	<profiles>
+		<profile>
```
Comment (bartoszmajsak) [inline pom.xml]: I'm not really sure if we need those profiles. We can simply run the build as follows:

```
$ mvn clean install -Darq.container.chameleon.configuration.chameleonTarget="jboss as:7.1.1.Final:managed"
```

File: src/test/resources/arquillian.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
     <container qualifier="dev" default="true">
```
Comment (bartoszmajsak) [inline src/test/resources/arquillian.xml]: For completeness, we can change `qualifier="chameleon"`.

**Comment by bartoszmajsak** [comment]
Thanks for your work Dipak!

Before we merge to upstream can you take care of these minor things as a part of this PR?

- [ ] Document in `README.md` how to run tests against different versions of WildFly
- [ ] Document in `README.md` the problem with `tools.jar` and how to run tests from IDE
- [ ] Format `pom.xml` with 4 spaces indentation, as I guess was done before?

File: pom.xml (hunk 1 above)
Comment (dipak-pawar) [inline pom.xml]: Jboss Server is starting. But it's hanging. Actual test execution is not happening even if we wait for more than 5 minutes.

Comment (bartoszmajsak) [inline pom.xml]: This is because you run it against JDK 8. I think we can drop this profile anyway.

**Comment by bartoszmajsak** [comment]
It seems that you have performed `push --force` or some other git kung-fu, and I cannot see your older commits (and cannot find old hashes). 

What I see now is that both Arquillain and Byteman versions are still the same as the old ones, which makes me wonder if that's an error or it wasn't there before and I've just missed that?

Anyway, I brought your changes upstream with @cbcc425 and missing versions update in @cbcc425. Thanks for your work.

**Pull request closed without merging**